solutions/security/get-started/content-connectors.md

```diff
@@ -0,0 +1,39 @@
+---
+applies_to:
+  stack: preview 9.1
+  serverless: preview
+---
+
+# Content connectors
+
+Elastic's content connectors allow you to extract, transform, index, and sync data from third-party applications including Github, PagerDuty, Jira, Teams, Google Drive, Slack, email, and more ([view all connectors](elasticsearch://reference/search-connectors/index.md)).
+
+{{stack}} supports two deployment methods for your connectors: Elastic managed, and self-managed. Self-managed connectors require you to deploy on your own infrastructure (for example, run some Python code on a server you manage). When you use an Elastic managed connector, Elastic runs the infrastructure for you. Self-managed connectors can be customized, whereas Elastic managed connectors do not allow customization. 
+
+{{sec-serverless}} only supports Elastic managed connectors. {{stack}} deployments support either self-managed or Elastic managed deployments.
```

Comment:
That's ok to leave out of scope of this issue for now, but I see here a good opportunity for a small comparison table to make it easier to understand the differences between both modes.
If you'd rather not do it right now, please create an issue :)

Reply:
The info should be good per https://github.com/elastic/docs-content/pull/2113#discussion_r2244750819 at least, but yeah a table might be better format